OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -45,7 +45,7 @@ public class AssetBrowserLogic : ChromeLogic
 		int currentFrame;
 
 		[ObjectCreator.UseCtor]
-		public AssetBrowserLogic(Widget widget, Action onExit, World world)
+		public AssetBrowserLogic(Widget widget, Action onExit, World world, Dictionary<string, MiniYaml> logicArgs)
```

Comment:
`Dictionary<string, string>` please.  The main reason for my earlier request was so that the chrome logic didn't need to deal with MiniYaml.